package.json

```diff
@@ -87,7 +87,7 @@
     "ember-cli-svgstore": "github:ciena-blueplanet/ember-cli-svgstore#977df1cf58ae43b1d98a591573c3e06947744321",
     "ember-computed-decorators": "0.3.0",
     "ember-concurrency": "0.7.19",
-    "ember-frost-core": "^8.0.0",
+    "ember-frost-core": "^8.2.2",
```

Comment:
This change should be reverted. It is not necessary to update the version of `ember-frost-core` here since it will already pick up the minor due to the `^` float.